frontend/src/metabase/common/components/QueryColumnPicker/BucketPickerPopover/BaseBucketPickerPopover.tsx

```diff
@@ -96,6 +96,7 @@ function _BaseBucketPickerPopover({
     <Popover opened={isOpened} position="right" onClose={handlePopoverClose}>
       <Popover.Target>
         <TriggerButton
+          className="base-bucket-picker-popover__trigger-button"
```

Comment:
:x: `:global` + BEM

Reply:
fixed by passing className to the TriggerButton